### DISCUSSION TITLE
How to stream audio from TV to other speakers via MA

### DISCUSSION THREAD

**Discussion by frr032** [music-assistant/support#2166] [Q&A] [answered]
How would I go about steaming TV audio from a Samsung smart frame TV QN32LS03CBF to other speakers using MA?  
Is it possible to stream the audio via the TV's LAN port or Optical port? What hardware would be required?

**Comment by OzGav** [comment, marked as answer]
Have a look atthe last entry here https://music-assistant.io/faq/stream-to/

Then maybe something from here https://www.google.com/search?q=optical+sound+to+usb
